scripts/pre_commit_linter.py

```diff
@@ -1463,6 +1471,107 @@ def _check_directive_scope(all_files):
     return summary_messages
 
 
+def _check_sorted_dependencies(all_files, parsed_js_files):
+    """This function checks that the dependencies which are
+    imported in the controllers/directives/factories in JS
+    files are in following pattern: dollar imports ,regular
```

Comment:
Bad comma placement, please fix:

```
files are in following pattern: dollar imports ,regular
```

Should be:

```
files are in following pattern: dollar imports, regular
```

Reply:
Thanks for picking out this!
Done.